src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -121,7 +121,15 @@ public function get($key, $default = null, $first = true)
         }
 
         if ($first) {
-            return \count($headers[$key]) ? (string) $headers[$key][0] : $default;
+            if (!\count($headers[$key])) {
```

Comment:
Let's remove the count() as it's useless anyway.

Reply:
Done :slightly_smiling_face: